packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -978,6 +978,16 @@ instance.userData.addExecuteFileCommand({
 asset.grantRead( instance.role );
 ```
 
+In addition to above the `MultipartUserData` can be used to change instance startup behavior. Multipart user data are composed
+from separate parts forming archive. The moment, and behavior of each part can be controlled with `Content-Type`, and it's wider
+than executing shell scripts.
+
+Some services (like AWS Batch) allows only `MultipartUserData`.
```

Comment:
I think a small example block showing how to create a multipart userdata and adding 2 parts to it might be helpful.

Can you think of a minimally useful example where someone would add 2 parts (that actually does something useful in a real AWS service?)

Reply:
Added example to increase maximum size of docker storage.